Library.ino

```diff
@@ -532,8 +535,8 @@ int Centre(int Length, int wid = AUTO, int Left = 0, int CHwid = 6) {
         bool     label_enabled    [PAGES][LABEL];     // Is label enabled?
         bool     label_visible    [PAGES][LABEL];     // Is label shown?
     int addlabel(int page, uint16_t X, uint16_t Y, String text, uint16_t textcolor,
-                 const GFXfont *font = NULL, byte textsizeX = 1, byte textsizeY = 1,
-                 bool enabled = true, bool visible = true) 
+                 const GFXfont *font, byte textsizeX , byte textsizeY ,
+                 bool enabled , bool visible = true) 

```

Comment:
Here, when I want to remove the default argument for `visible`, the compiler says `GFXfont does not name a type` but will compile with the default argument.

Reply:
Maybe the reason is:

1. Arduino builder detects the function and it's prototype as two different functions,
2. Arduino builder places another prototype for the function
3. The new prototype is before `#include <Adafruit_GFX.h>`
4. `GFXfont` is not declared before that line
5. C++ compiler throws an error